startup.py

```diff
@@ -406,3 +451,26 @@ def _synergy_config_files(config_match=None):
                     synergy_configs.append(search_path)
 
     return synergy_configs
+
+def _context_entity_type_id(context):
```

Comment:
i missing a verb at the start of the method name. How about `_extract_entity_from_context` or something like that?